1-callbacks/solution.js

```diff
@@ -31,17 +31,50 @@ node solution.js name1 name2 name3
 5. another challenge is: after you solve the challenge using callback style, in another file promisify the callback and solve it again
 ** give a look to node.js util.promisify, avoid to alter the validate-user.file **
 */
+import validateUser from "./validate-user.js";
```

Comment:
We are using CommonJS module, so we should to use `require`.
```suggestion
const validateUser = require("./validate-user");
```